index.js

```diff
@@ -1,17 +1,57 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = 0;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.length += 1;
```

Comment:
Es mejor usar el propio array de items! asi siempre te aseguras que tienes esos elementos!
```suggestion
    this.length += this.items.length;
```